### ISSUE TITLE
The Project Uploads API is throwing 520 Errors

### ISSUE THREAD

**Issue by AzonInc** [jdalrymple/gitbeaker#2843]
**Description**

- Node.js version: 17.5.0
- Gitbeaker version: 35.7.0
- Gitbeaker release (cli, node, browser, core, requester-utils): node
- OS & version: Windows 10

The Project Uploads API is not working properly and throwing 520 Errors.

**Steps to reproduce**

I use the following Code to upload a file:
```Javascript
let filename = 'test.txt';
let file_buffer = fs.createReadStream(filename);

gitlab_api.Projects.upload(xxxxxxxxxxx, file_buffer, {
    metadata: {
        filename: filename,
        contentType: 'text/plain'
    }
}).then(console.log).catch(console.log);
```

It's working with axios and the code:
```Javascript
let filename = 'test.txt';
let file_buffer = fs.createReadStream(filename);

var fData = new FormData();
fData.append("file", file_buffer);

axios.post(store.gitlab_url + "api/v4/projects/xxxxxxxxxxx/uploads", fData, { headers: { ...fData.getHeaders(), 'PRIVATE-TOKEN': xxxxxxxxxxxxxxxxxxxxx } }).then(console.log).catch(console.log);
```

**Expected behaviour**

Show uploaded File Data

**Actual behaviour**

Throwing 520 Error:
```Javascript
HTTPError: Response code 520 (undefined)
    at Request.<anonymous> (C:\Users\flori\Desktop\restaurant-companion\node_modules\got\dist\source\as-promise\index.js:118:42)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  code: 'ERR_NON_2XX_3XX_RESPONSE',
  timings: {
    start: 1669058768824,
    socket: 1669058768825,
    lookup: 1669058768827,
    connect: 1669058768846,
    secureConnect: 1669058768872,
    upload: 1669058768891,
    response: 1669058770149,
    end: 1669058770150,
    error: undefined,
    abort: 1669058770150,
    phases: {
      wait: 1,
      dns: 2,
      tcp: 19,
      tls: 26,
      request: 19,
      firstByte: 1258,
      download: 1,
      total: 1326
    }
  }
}
```

**Possible fixes**

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
Once the latest release is published, here is a sample:

```js
import { Gitlab } from '@gitbeaker/rest'
import FS from 'node:fs/promises'

const data = await FS.readFile('./README.md')
const file = new Blob([data], { type: 'text/markdown' });

const api = new Gitlab({
  token
})

api.Projects.upload(12345, file, { filename: 'README.md' })
.then(r => {
  console.log(r)
})
```

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`37.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/37.0.0) :rocket:
